.github/actions/setup-spack-stack/action.yaml

```diff
@@ -247,21 +249,17 @@ runs:
     run: |
       if [[ ! "${{ inputs.templates }}" == "empty" ]]; then
         cd ${{ inputs.path }}
-        if [[ "$RUNNER_OS" == "Linux" ]]; then
-          # Not yet: need lmod installed, loaded and module use command
-          echo ""
-        elif [[ "$RUNNER_OS" == "macOS" ]]; then
-          source /usr/local/opt/lmod/init/profile
-          module use $PWD/envs/${{ inputs.name }}/install/modulefiles/Core
-          # This needs more work: need to replace "${{ inputs.compiler }}" = "gcc@9"
-          # with "gcc/9", otherwise the module load command will fail.
-          # module load stack-${{ inputs.compiler }}
-          # module load stack-python
-          # # jedi-tools doesn't have any mpi dependencies
-          # if [[ ! "${{ inputs.specs }}" == "jedi-tools" ]]; then
-          #   module load stack-${{ inputs.mpi }}
-          # fi
-        fi
+        #if [[ "$RUNNER_OS" == "Linux" ]]; then
```

Comment:
This is just commenting out the entire block 250-262, doesn't make a difference.